WebApplication/Backend/Services/PrescriptionService.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Model.MedicalExam;
+using System;
+using System.Collections.Generic;
+using WebApplication.Backend.DTO;
+using WebApplication.Backend.Repositorys;
+
+namespace WebApplication.Backend.Services
+{
+    public class PrescriptionService
+    {
+        private PrescriptionRepository prescriptionRepository;
+        private IPrescriptionRepository iPrescriptionRepository;
+        public PrescriptionService()
+        {
+            this.prescriptionRepository = new PrescriptionRepository();
+            this.iPrescriptionRepository = new PrescriptionRepository();
```

Comment:
You have 2 identical fields, the type should be IPerscriptionRepository but it can be initialized as PerscriptionRepository through dependency injection through constructor